client/e2e/pages/EventsPage.ts

```diff
@@ -91,9 +98,13 @@ export class EventsPage {
 
     this.deleteEventButton = page.getByTestId('event-delete-button');
 
+    this.eventContinue = page.getByRole('button', {name: 'Continue'});
+    this.eventClose = page.getByRole('button', {name: 'Close', exact: true});
     this.eventContinue = page.getByRole('button', {name: 'Continue'});
     this.eventClose = page.getByRole('button', {name: 'Close', exact: true});
 
+    this.eventOption1 = page.getByRole('button', {name: 'Default'});
+    this.eventOption2 = page.getByText('Active:', {exact: true});
     this.eventOption1 = page.getByRole('button', {name: 'Default'});
     this.eventOption2 = page.getByText('Active:', {exact: true});

```

Comment:
Duplicate code here. EventOption naming can be improved to reflect functionality.